app/azmet-shiny-template.html

```diff
@@ -0,0 +1,384 @@
+                            <div class="node__content clearfix">
+                              <div class="field field--name-field-az-main-content field--type-entity-reference-revisions field--label-hidden field__items">
+                                <div class="field__item">
+                                  <div class="mb-5 paragraph background-wrapper paragraph--type--az-text-background paragraph--view-mode--default bg-triangles-top-left bg-azurite py-5 full-width-background">
+                                    <div class="container">
+                                      <div class="az-full-width-row">
+                                        <div class="az-full-width-column-content">
+                                          <div class="content">
+                                            <div class="clearfix text-formatted field field--name-field-az-text-area field--type-text-long field--label-hidden field__item">
+                                              <div class="d-flex flex-nowrap">
+                                                <div class="icon-circle d-flex justify-content-center align-items-center mr-3">
+                                                  <span class="lm-wrench"></span> <!-- Edit this line based on individual Shiny app --> <!-- icon list at: https://staging.azmet.arizona.edu/modules/custom/azmet_views/fonts/icomoon/style.css -->
+                                                </div>
+                                                <div class="titles">
+                                                  <h2 class="mt-4 d-inline">Data Preview and Download</h2> <!-- Edit this line based on individual Shiny app -->
+                                                  <h5 class="mb-0 mt-2">Preview and download hourly and daily data by station and date range</h5> <!-- Edit this line based on individual Shiny app -->
+                                                </div>
+                                              </div>
+                                            </div>
+                                          </div>
+                                        </div>
+                                      </div>
+                                    </div>
+                                  </div>
+                                </div>
+                              </div>
+                            </div>
+                          </article>
+                        </div>
+                      </div>
+                    </section>
+                  </main>
+                </div>
+              </div>
+            </div>
+            <div class="container">
+              <!-- div -->
+                <!-- Shiny app -->
+                {{sidebarLayout}}
```

Comment:
```suggestion
                {{ sidebarLayout }}
```
Just a style suggestion, based on the Shiny template documentation

Reply:
This basically just sticks the entire Shiny app in in this place?  That's super nice!